### PR TITLE
Document wp-admin/js/tags.js

### LINKED ISSUE #15: Document wp-admin/js/tags.js

### PR DESCRIPTION
**Pull request by atimmer**
Fixes #15

### CODE REVIEW

File: src/wp-admin/js/tags.js [hunk 1]
```diff
@@ -2,38 +2,64 @@
```
Comment (rensw90) [inline src/wp-admin/js/tags.js]: Missing file header: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#file-headers

File: src/wp-admin/js/tags.js [hunk 2]
```diff
@@ -2,38 +2,64 @@
 
 jQuery(document).ready(function($) {
 
+	/**
+	 * Adds an event handler to the delete term link on the term overview page. Cancels default event handling and event
+	 * bubbling.
+	 */
```
Comment (rensw90) [inline src/wp-admin/js/tags.js]: Maybe add @listens

File: src/wp-admin/js/tags.js [hunk 3]
```diff
@@ -2,38 +2,64 @@
...
+
 		if ( 'undefined' != showNotice )
 			r = showNotice.warn();
+
 		if ( r ) {
 			data = t.attr('href').replace(/[^?]*\?/, '').replace(/action=delete/, 'action=delete-tag');
+
+			/**
+			 * Does a request to the server to delete the term for which the user clicked on the delete term link.
+			 *
```
Comment (rensw90) [inline src/wp-admin/js/tags.js]: Weird sentence.

File: src/wp-admin/js/tags.js [hunk 4]
```diff
@@ -2,38 +2,64 @@
...
+			 * @param {string} r The response from the server.
+			 */
 			$.post(ajaxurl, data, function(r){
 				if ( '1' == r ) {
 					$('#ajax-response').empty();
 					tr.fadeOut('normal', function(){ tr.remove(); });
+
 					// Remove the term from the parent box and tag cloud
 					$('select#parent option[value="' + data.match(/tag_ID=(\d+)/)[1] + '"]').remove();
```
Comment (rensw90) [inline src/wp-admin/js/tags.js]: ?

File: src/wp-admin/js/tags.js [hunk 5]
```diff
@@ -2,38 +2,64 @@
...
+			/**
+			 * Does a request to the server to delete the term for which the user clicked on the delete term link.
+			 *
+			 * @param {string} r The response from the server.
+			 */
 			$.post(ajaxurl, data, function(r){
 				if ( '1' == r ) {
 					$('#ajax-response').empty();
 					tr.fadeOut('normal', function(){ tr.remove(); });
+
```
Comment (rensw90) [inline src/wp-admin/js/tags.js]: Don't understand what this means

File: src/wp-admin/js/tags.js (hunk 4 above)
Comment (rensw90) [inline src/wp-admin/js/tags.js]: Don't understand what this means. It would be nice if you add some context.

File: src/wp-admin/js/tags.js [hunk 6]
```diff
@@ -2,38 +2,64 @@
...
 			tr.children().css('backgroundColor', '#f33');
 		}
+
 		return false;
 	});
 
+	/**
+	 * Adds an event handler to the form submit on the term overview page. Cancels default event handling and event
+	 * bubbling.
+	 */
```
Comment (rensw90) [inline src/wp-admin/js/tags.js]: Add @listens

File: src/wp-admin/js/tags.js [hunk 7]
```diff
@@ -2,38 +2,64 @@
...
 		var form = $(this).parents('form');
 
 		if ( ! validateForm( form ) )
 			return false;
 
+		/**
+		 * Does a request to the server to add a new term to the database
+		 * 
+		 * @param {string} r The response from the server.
+		 */
```
Comment (rensw90) [inline src/wp-admin/js/tags.js]: Doesn't this return something?

**Comment by terw-dan** [comment]
CR done

**Comment by moorscode** [comment]
@atimmer needs CR to be applied

**Comment by atimmer** [comment]
Processed CR.

**Comment by moorscode** [comment]
A new function has been added, which now requires documentation as well.

**Comment by moorscode** [comment]
Patch submitted https://core.trac.wordpress.org/ticket/41069

**Comment by atimmer** [comment]
Fixed in core.

**Pull request closed without merging**